docs/src/pages/[platform]/connected-components/account-settings/account-settings.react.mdx

```diff
@@ -12,6 +12,10 @@ Account Settings components are a set of standalone components that add [user ma
 
 ## Quick Start
 
+<Alert variation="info" role="none">
+Next.js 13.4+ introduces [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). To use Amplify UI componenents, you must use them inside the Client Component tree by placing `"use client"` at the boundary between your Client and Server component module graph. If you are using it, you need to do some extra setup [here](/react/getting-started/usage/nextjs#connected-components).
```

Comment:
Can we please move this to a shared fragment so we only have to update it in one place?

Reply:
Will do